integration-tests/src/test/java/oracle/kubernetes/operator/ItOperatorUpgrade.java

```diff
@@ -293,6 +202,29 @@ private void checkDomainRollingRestarted() throws Exception {
     }
   }
 
+  /**
+   * Check whether the weblogic server instances are still RUNNING 
+   * not restarted due to Operator Upgrade by comparing the creationTimestamp
+   * before and after upgrade
+   *
+   * @throws Exception If restarted
+   */
+  private void checkDomainNotRestarted() throws Exception {
+    TestUtils.checkPodReady(DUID + "-" + domain.getAdminServerName(), DOM_NS);
+    for (int i = 2; i >= 1; i--) {
+     TestUtils.checkPodReady(DUID + "-managed-server" + i, DOM_NS);
+    }
+    String m1 = TestUtils.getCreationTimeStamp(DOM_NS,DUID + "-managed-server1");
+    String m2 = TestUtils.getCreationTimeStamp(DOM_NS,DUID + "-managed-server2");
+    String admin = TestUtils.getCreationTimeStamp(DOM_NS,DUID + "-admin-server");
+    System.out.println("After Upgrade M1CTS[" + m1 + "]");
```

Comment:
log not sop?

Reply:
what's an M1CTS?  think of the person reading this log message, who is probably not you. how do they know what this means?

Reply:
Modified the Log Message as follows ...
println("After Upgrade CreateTimeStamp for managed-server1 pod [" + m1 + "]");
println ("After Upgrade CreateTimeStamp for managed-server2 pod [" + m2 + "]");
println("After Upgrade CreateTimeStamp for admin-server pod [" + admin + "]");